### ISSUE TITLE
address cache ADDRESS -e vs address cache ADDRESS -e -f

### ISSUE THREAD

**Issue by buhtignew** [slimcoin-project/pacli#217]
I've run `address cache mmBwLVyqZWuUDuTpmmpzBvT41azKBevZhh -e` and got:
```
Deleting block locator entries of addresses: ['mmBwLVyqZWuUDuTpmmpzBvT41azKBevZhh']
This is a dry run. Use --force to really erase the entry of this address.
```
Then I've run `address cache mmBwLVyqZWuUDuTpmmpzBvT41azKBevZhh -e -f` and got:
```
Deleting block locator entries of addresses: ['mmBwLVyqZWuUDuTpmmpzBvT41azKBevZhh']

Error: Address not found.
```
So the impression here is that during the dry run the code haven't really done anything, since the fact that the address were not there could've been outputted at the stage of dry run.
Or maybe I'm not fully understanding the concept of dry run in this case?

**Comment by d5000** [comment]
Yes this dry run was too short imo too. I changed it now so the error will be shown in the case the address doesn't exist in the block locator file.

Commit: 622e652
